picard/util/imagelist.py

```diff
@@ -104,42 +119,46 @@ def __init__(self):
         self.update_orig_metadata = False
 
 
-def _process_images(state, src_obj):
-    from picard.track import Track
-
+def _process_images(state, src_obj, Track):
     # Check new images
     if state.update_new_metadata:
-        if state.new_images != set(src_obj.metadata.images):
+        src_dict = src_obj.metadata.images.hash_dict()
+        og_len = len(state.new_images)
```

Comment:
s/og_len/orig_len/?

Reply:
Changed for the `s`. I didn't want to use `orig_len` because there are `metadata` and `orig_metadata`.

Reply:
Actually `s` is even less explicit (and `s` is used much more often for strings than lengths, if we were to keep one-char variable name, `l` would be more logicial).
What's `len(state.new_images)` ?